mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -244,6 +244,12 @@ def preprocess(ds, calendar, simulationStartTime, timeVariableName,
     and Xylar Asay-Davis
     """
 
+    # following a suggestion by @rabernat
+    # https://github.com/pydata/xarray/issues/2064#issuecomment-381717472
+    concat_dim = 'Time'
+    coord_vars = [v for v in ds.data_vars if concat_dim not in ds[v].dims]
```

Comment:
Do we want to include the drop option here, just default it to `False`?   I'm thinking we may want to have it as Ryan suggested in the following snipet

```
    if drop:
        return ds.drop(coord_vars)
    else:
        return ds.set_coords(coord_vars)
```